test/unittest/tests/utils_extlist.h

```diff
@@ -15,32 +15,32 @@
 
 namespace utils {
 /*!
- * An offset (_off) and a size (_size) of a WT_EXT for two use cases: specifying a test setup or an
+ * An offset (off) and a size (size) of a WT_EXT for two use cases: specifying a test setup or an
  * expected result.
  */
 struct off_size {
-    wt_off_t _off;
-    wt_off_t _size;
+    wt_off_t off;
+    wt_off_t size;
 
-    off_size(wt_off_t off = 0, wt_off_t size = 0) : _off(off), _size(size) {}
+    off_size(wt_off_t off = 0, wt_off_t size = 0) : off(off), size(size) {}
     /*!
      * end --
-     *     Return the end of the closed interval represented by _off and _size.
+     *     Return the end of the closed interval represented by off and size.
      */
     wt_off_t
     end(void) const
```

Comment:
We often don't do this, but I also like to label such functions with `noexcept` :)

Reply:
Done